src/bin/build.rs

```diff
@@ -90,10 +93,12 @@ pub fn execute(options: Options, config: &Config) -> CliResult {
 
     let root = find_root_manifest_for_wd(options.flag_manifest_path, config.cwd())?;
 
-    let spec = if options.flag_all {
-        Packages::All
-    } else {
-        Packages::Packages(&options.flag_package)
+    let spec = match (options.flag_all, &options.flag_exclude) {
```

Comment:
This `match` would be used in many subcommands. May I place it in `cargo::util`?